app/src/androidTest/java/com/android/sample/ui/components/AttendantPreviewTest.kt

```diff
@@ -0,0 +1,68 @@
+package com.android.sample.ui.components
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.android.sample.model.image.ImageRepositoryFirestore
+import com.android.sample.model.image.ImageViewModel
+import com.android.sample.model.profile.User
+import com.android.sample.resources.dummydata.simpleUser
+import com.android.sample.resources.dummydata.testUser
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+
+class AttendantPreviewTest {
```

Comment:
Very good test class, you test a lot of possibilities, I think that this is quite complete